### ISSUE TITLE
May be user error but the equations do not work forwards and backwards the same

### ISSUE THREAD

**Issue by Assimilater** [aritchie05/EcoCraftingTool#35]
![image](https://user-images.githubusercontent.com/689486/162881141-fccd8acc-91e9-42fe-bbea-e9c127422722.png)
Consider the following image for the forward equation to determine the price of a recurve bow

I get 63.57

Here is the reverse when looking up recurve bow specifically

![image](https://user-images.githubusercontent.com/689486/162881266-c598384a-3ac9-4c66-8611-2d9c877f7541.png)
the 6.3 is rounded from 6.4 which is what I got in the first screenshot (not shown)

**Comment by aritchie05** [comment]
Good question, the detail at play here is that the profit percentage only applies to the output price for a final output. Each output also stores a "base price" which is the cost of production before profit. If an output material is used for further recipes, the base price of that output is used rather than the profit price.

I did a quick check and it looks like the base price of steel production for your first screenshot is 3.67. Plugging in that value for steel input cost and we see the 63.57 result price.

![image](https://user-images.githubusercontent.com/15224019/162884623-7f5f4605-cc94-4ed7-b2da-c37af362f217.png)

The reason I did it this way is because production chains that involve many steps in the process (e.g. mechanics final products) can display very misleading production costs if profit % is applied at every step in the process.
